src/bootstrap/dist.rs

```diff
@@ -483,7 +483,7 @@ impl Step for Rustc {
 
             // Copy runtime DLLs needed by the compiler
             if libdir != "bin" {
```

Comment:
I think this check is presumably wrong now, and we should be checking if the final component of rustc_libdir is "bin"?

Reply:
Looks reasonable, I'll try it.

Reply:
Changed.